modules/mux/targets/host/source/device.cpp

```diff
@@ -360,10 +360,16 @@ device_info_s::device_info_s(host::arch arch, host::os os, bool native,
     case host::arch::ARM:
     case host::arch::X86:
       this->address_capabilities |= mux_address_capabilities_bits32;
+      this->atomic_capabilities |= mux_atomic_capabilities_8bit |
```

Comment:
I can't see this initialized anywhere, but you are using |=

Reply:
You're right, this should have been `=`, fixed and likewise for below.